src/api/apiClient.ts

```diff
@@ -0,0 +1,35 @@
+import axios, { AxiosError } from 'axios'
+import { BASE_URL } from './common'
+
+export const apiClient = axios.create({
+    baseURL: `${BASE_URL}`,
```

Comment:
tiny thing, but this doesn't need to be a string template
```suggestion
    baseURL: BASE_URL,
```